Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py

```diff
@@ -547,23 +523,19 @@ def test_module(client: Client, params: Dict[str, Any], first_fetch_time: int) -
     # Cortex XSOAR will print everything you return different than 'ok' as
     # an error
     try:
-        if params.get('isFetch'):  # Tests fetch incident:
-            alert_status = params.get('alert_status', None)
-            alert_type = params.get('alert_type', None)
-            min_severity = params.get('min_severity', None)
-
+        time = dateparser.parse('1 minute')
+        assert time
```

Comment:
Only because it is for learning purposes, in the test module we don’t want to run on big time windows. In my code it is a bit redundant as I generate according to max fetch but in real API it might cause the test-module to run for long time.